api/src/main/java/edu/cornell/mannlib/vitro/webapp/beans/IndividualImpl.java

```diff
@@ -20,6 +20,7 @@
 /**
  * Represents a single entity record.
 */
+@SuppressWarnings("ComparableType")
```

Comment:
@grahamtriggs is this suppressed because it is a false positive (per errorprone [documentation](https://errorprone.info/bugpattern/ComparableType)) or because it's not a simple fix to pass the errorprone check?

Reply:
I'm treating it as a false positive, because it is using an interface that the IndividualImpl implements. Therefore, you would expect that when the comparable interface is being used, it is used to compare lists containing Individual, which could be any implementation.

It's reasonable to expect that we have lists of Individual (the interface), and that we would want to compare them, regardless of the underlying implementation.

For the two SearchResult occurrences, they are only used in Lists containing only the concrete class, and in any case, suggesting that they can be compared to any Object is too vague.